functional-tests/fn_bridge_deposit.py

```diff
@@ -95,6 +101,4 @@ def do(self, ctx: flexitest.RunContext, evm_addr: str):
 
         balance = int(rethrpc.eth_getBalance(f"0x{evm_addr}"), 16)
         net_balance = balance - original_balance
-        assert (
-            net_balance == deposit_amount
-        ), f"deposit processed multiple times, extra: {balance - original_balance - deposit_amount}"
+        assert balance == deposit_amt * SATS_TO_WEI, f"deposit processed multiple times: {balance}"
```

Comment:
This kinda seems like it's duplicating the above checks?  Should specify that that was the amount accepted and what the expected amount was.